src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -401,7 +441,7 @@ class ActiveFocusInvestigation extends React.Component<
               columns: emptyPlansColumns,
             };
             routineReactivePlans.push(
-              <NullDataTable tableProps={tableProps} reasonType={header} key={`${'current'}`} />
+              <NullDataTable tableProps={tableProps} reasonType={header} key={i} />
```

Comment:
why was this done?

Reply:
If you have more than two nullable tables they both have the same key